staging/src/k8s.io/apiextensions-apiserver/pkg/controller/openapi/builder/builder_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package openapi
+package builder
```

Comment:
could it be `builder_test`?

Reply:
it could. But we never do that.

Reply:
why? I think that "black-box" testing is better because the tests have access only to the exported methods/fields.

Reply:
it's better. Just not common in this repo. Might be a simple rename here though.

Reply:
we have whitebox tests in there. So it's more than a rename and out of scope for this PR.